_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -30,11 +25,10 @@ This rule applies to any [visible][] HTML element that is a [semantic link][], f
 
 ## Expectation
 
-For each test target the [outcome][] of one of the following rules is "passed":
-
-- [Inline link has either distinguishable style not based on color (hue) alone or content that indicates it is a link](https://act-rules.github.io/rules/88407d)
-- [Inline link has distinguishable style or content on hover and focus](https://act-rules.github.io/rules/548868)
+There exists at least one [inclusive descendant element][] and one [adequate inspection state][] of the test target such that at least one of the following is true for all [interaction states][]:
```

Comment:
🤔 The way I read this, it must be the same descendant, inspection state and condition that is true for all interaction states.

So:
* has a link icon when not visited, underlined when visited
* underlined when not visited but focused; underlined when visited and not focused

would both fail the rule.

These ure both fairly bad design (I say), but I am not sure this really a WCAG failure.
If we want to allow the element to pass in various ways depending on the interaction state, we need to put the universal quantifier first:
> for all interaction states, there exist at least …